entries/g/gloriousgaming.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Glorious": {
+    "domain": "gloriousgaming.com",
+    "contact": {
+      "facebook": "GloriousPCG",
```

Comment:
```suggestion
```

The check is failing

Reply:
Looks like Facebook is restricting access to some pages more than others.
We might need a Facebook API key in the future :unamused: 
In the meantime I second @hkamran80's suggestion ^

Reply:
The page exists though and you can access it (like some of the `domain` changes I have been working on lately). Is there something that may need to be updated with the tests?

<img width="512" alt="Screen Shot 2023-02-02 at 2 55 22 AM" src="https://user-images.githubusercontent.com/45466974/216307047-c76b65af-5ed1-4bc0-bc20-397a1d8ac26a.png">

Reply:
Like @Carlgo11 said, we might need to add an API key for Facebook. This removal is temporary, at least until we can fix this bug.